wagtail_localize_rws_languagecloud/templates/wagtailadmin/pages/side_panels/includes/page_locale_status.html

```diff
@@ -0,0 +1,31 @@
+{% extends "wagtailadmin/pages/side_panels/includes/page_locale_status.html" %}
+{% load i18n wagtailadmin_tags wagtaillocalizerwslanguagecloud_tags %}
+
+<link rel="stylesheet" href="{% versioned_static 'wagtail_localize_rws_languagecloud/css/wagtail-localize-overrides.css' %}" type="text/css">
+
+{% block action %}{# Wagtail 3.0 #}
```

Comment:
What's this comment telling me?

Reply:
`page_locale_status.html` is used in 3.0, while `shared/side_panels/includes/status/locale.html` in 4+

Reply:
made the comments explicit